app/guid-file/controller.ts

```diff
@@ -180,7 +193,131 @@ export default class GuidFile extends Controller {
 
     @action
     versionChange(version: number) {
-        this.set('revision', +version);
+        this.set('revision', version);
+    }
+
+    @action
+    async toggleVersions() {
```

Comment:
I was envisioning more of an await here since we would want the slide transition dependent on the toggle but can remove it if you think it would process first regardless.